File: framework/src/main/scala/com/software/dm/swallow/stormy/scala/security/addrlib/bdia/bean/T_basic_type_rel.scala
package com.software.dm.swallow.stormy.scala
package security.addrlib.bdia.bean

/**
  *@author darlingming@126.com
  *2018-05-01
  */
class T_basic_type_rel extends  Serializable {
  private var n_id = 0L
  private var type_id = ""
  private var all_name = ""

  def this(n_id: Long, type_id: String, all_name: String) {
    this()
    this.n_id = n_id
    this.type_id = type_id
    this.all_name = all_name
  }

  def getN_id: Long = n_id

  def getType_id: String = type_id

  def getAll_name: String = all_name
}

File: framework/src/main/java/com/software/dm/swallow/stormy/security/match/bdia/bean/T_basic_type_rel.java
package com.software.dm.swallow.stormy.security.match.bdia.bean;

import java.io.Serializable;

/**
 * @author DM
 * @version v1.0.0.1
 * @Description
 * @date 2017
 */
public final class T_basic_type_rel implements Serializable {

    private long n_id;
    private String type_id;
    private String all_name;

    public T_basic_type_rel(long n_id, String type_id, String all_name) {

        this.n_id = n_id;
        this.type_id = type_id;
        this.all_name = all_name;
    }

    public long getN_id() {
        return n_id;
    }

    public String getType_id() {
        return type_id;
    }

    public String getAll_name() {
        return all_name;
    }

}
